packages/calendar-utils/src/calendar-body/calendar-body.styles.ts

```diff
@@ -165,9 +188,9 @@ const getInputContainerStyles = (props, state) => {
   `;
 };
 
-const getDateTimeInputStyles = (props) => {
+const getDateTimeInputStyles = (props: TCalendarBody) => {
   const baseStyles = [
-    getInputStyles(props),
+    getInputStyles(props, customProperties),
```

Comment:
I think we should make the `theme` option optional in the `getInputStyles` function.